components/automate-ui/src/app/components/admin-sidebar/admin-sidebar.component.html

```diff
@@ -27,7 +27,7 @@
     </app-authorized>
 
     <div *ngIf="(iamMajorVersion$ | async) !== 'v1'">
-      <div class="group" *ngIf="policies.visible || roles.visible || projects.visible">Access Management</div>
+      <div class="group" *ngIf="policies.visible || roles.visible || projects.visible && (iamMinorVersion$ | async) === 'v1'">Access Management</div>
```

Comment:
Sorry, got distracted and forgot one comment.
I would urge adding parentheses around the new expression `(projects.visible && (iamMinorVersion$ | async) === 'v1'")`. Should not rely on assumptions about operator precedence.